app/javascript/components/froggo-dropdown.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="relative" v-on-clickaway="closeDropdown">
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'v-on-clickaway' should be on a new line   vue/max-attributes-per-line